gprofiler/client.py

```diff
@@ -242,6 +242,6 @@ def submit_spark_metrics(self, snapshot: MetricsSnapshot) -> Dict:
 def bake_metrics_payload(snapshot: MetricsSnapshot) -> Dict[str, Any]:
     return {
         "format_version": 0,
-        "timestamp": int(snapshot.timestamp.timestamp()),
+        "timestamp": format_as_short_iso8601(snapshot.timestamp),
```

Comment:
As discussed let's change it to the format of `+00:00` instead of `Z`.

Reply:
i.e `get_iso8601_format_time`

Reply:
fixed.